nbextensions/appCell2/widgets/tabs/logTab.js

```diff
@@ -1,15 +1,14 @@
-/**
- * This is the entrypoint module for the job status / log viewer tab of the app cell.
- */
-define(['bluebird', 'common/html', 'common/ui', 'util/jobLogViewer'], (
-    Promise,
-    html,
-    UI,
-    JobLogViewerModule
-) => {
+define([
+    'bluebird',
+    'common/html',
+    'common/ui',
+    'common/jobStateViewer',
+    'common/simpleLogViewer',
+], (Promise, html, UI, JobStateViewerModule, SimpleLogViewerModule) => {
     'use strict';
 
-    const { JobLogViewer } = JobLogViewerModule;
+    const { JobStateViewer } = JobStateViewerModule;
```

Comment:
use the new job state and job log viewer modules as we now have the job manager in charge of all things job.